### DISCUSSION TITLE
.NET Maui

### DISCUSSION THREAD

**Discussion by CSharpProgrammerA** [linq2db/linq2db#4679] [Q&A]
Can I use Linq2db in .NET Maui ?
Thanks
Andre

**Comment by igor-tkachev** [comment]
Yes, you can use linq2db in .NET MAUI, but it also depends on the data provider. If the provider supports MAUI (e.g., SQLite, PostgreSQL via Npgsql, or MySQL via MySqlConnector), then it should work without issues. The key requirement is that the provider itself is compatible with the MAUI environment (Android, iOS, Windows, etc.).
